modules/retail/pages/retail-admin.adoc

```diff
@@ -2,4 +2,4 @@
 = Administration
 
 This sections contains notes on administering your {smr} installation.
-For general administration tasks, see the {susemgr} documentation at https://www.suse.com/documentation/suse-manager-3/
+For general administration tasks, see the {susemgr} documentation at https://www.suse.com/documentation/suse-manager-4.
```

Comment:
```suggestion
For general administration tasks, see the {susemgr} documentation at https://www.suse.com/documentation/suse-manager.
```
That way it will always redirect to the latest.